js/index.js

```diff
@@ -40,3 +40,88 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+const bigCircle=document.getElementById('cta-img');
+bigCircle.setAttribute('src', siteContent['cta']['img-src']);
+
+const banner=document.getElementById('middle-img');
+banner.setAttribute('src', siteContent['main-content']['middle-img-src']);
+
+
+const navItems=document.querySelectorAll('nav a');
+navItems[0].textContent=siteContent['nav']['nav-item-1'];
```

Comment:
If you wanna push yourself, Try to apply the values with a for loop instead of choosing each index individually. Try to reference your `forEach` that you did with colors